CONTRIBUTING.md

```diff
@@ -400,6 +400,10 @@ class...
   (example: sale_order_line_ids)
 * `Many2One` fields should have `_id` as suffix
   (example: partner_id, user_id, ...)
+* If the name of field is the same to name of label string don't setting `string` parameter for fields of api v8.
```

Comment:
Rephrase this to:

`If the technical name of the field (the variable name) is the same to the string of the label, don't put `string` parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized`